thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/dashboard/resources/v2/anomalies/AnomalySearcher.java

```diff
@@ -70,7 +70,11 @@ public AnomalySearcher() {
    * @return the result
    */
   public Map<String, Object> search(AnomalySearchFilter searchFilter, int limit, int offset) {
-    Predicate predicate = Predicate.EQ("child", false);
+    Predicate predicate = Predicate.NEQ("baseId", 0);
```

Comment:
what's the rational behind querying by baseId? Can you also include it in the code-comment.

Reply:
+1

Reply:
thanks, added the comment